tests/lightning_tensor/test_lightning_tensor.py

```diff
@@ -90,8 +90,12 @@ def test_invalid_wires_none(self, method):
 
     def test_invalid_cutoff_mode(self, method):
         """Test that an error is raised if an invalid cutoff mode is provided."""
-        with pytest.raises(ValueError):
-            LightningTensor(wires=2, cutoff_mode="invalid_mode", **method)
+        if method["method"] == "mps":
+            with pytest.raises(ValueError):
+                LightningTensor(wires=2, cutoff_mode="invalid_mode", **method)
+        if method["method"] == "tn":
+            with pytest.raises(TypeError):
+                LightningTensor(wires=2, cutoff_mode="invalid_mode", **method)
```

Comment:
Why this is a ValueError for `mps` and a TypeError for `tn`.
Could we maybe have the same type of error?

Reply:
Good question! It should be a `TypeError` instead of `ValueError` for `tn` since `cutoff_mode` arg is not accepted in `tn` and is exclusively for `svd` decomposition for the `MPS`.